roles/openshift_master_upgrade/handlers/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
How much crazy is to create another ``openshift_master_restart`` role that would provide the handlers. The ``openshift_master`` and ``openshift_master_upgrade`` would then depend on it.